packages/arb-util/core/lookup.go

```diff
@@ -163,7 +164,7 @@ func waitForMessages(ctx context.Context, db ArbCoreInbox) (MessageStatus, error
 		if status == MessagesEmpty {
 			return 0, errors.New("should have messages")
 		}
-		if status != MessagesReady {
```

Comment:
Do we need to handle the MessagesError case here?

Reply:
`MessagesStatus` always returns an `error` object when there is an error, so it is unneccesary.